app/views/dashboard/_node_meta.html.erb

```diff
@@ -25,7 +25,11 @@
 </div>
 
 <div class="card-meta-links">
+<<<<<<< HEAD
```

Comment:
need to remove this.
Same in lines 30,32.

Reply:
Should I make another PR?

Reply:
Just push another commit reflecting these changes.

Reply:
Sorry, I didn't see your message. I will change this now.

Reply:
Sir, I have updated the file.